cmd/check_spec.go

```diff
@@ -29,9 +29,22 @@ var (
 	checkSpecCmd = &cobra.Command{
 		Use:   "check:spec",
 		Short: "Check a Rosetta implementation satisfies Rosetta spec",
-		Long: `Detailed Rosetta spec can be found in https://www.rosetta-api.org/docs/Reference.html.
-			Specifically, check:spec will examine the response from all data and construction API endpoints,
-			and verifiy they have required fields and the values are properly populated and formatted.`,
+		Long: `Check:spec checks if a Rosetta implementation satisfies either Coinbase specific requirements, or
+minimum requirements specified in rosetta-api.org. 
+		
+The minimum requirements mostly verify if a API response has required fields and if those fields are correctly 
+formatted with proper value, e.g. the response of /network/list should contain a list of network identifiers.
+		
+The Coinbase specific requirements may not be documented in rosetta-api.org, but they should be satisfied
+because otherwise the issues will come up when we integrate an asset with Coinbase platform.
+		
+Here are a few examples of Coinbase spec requirements:
+1. The network_identifier in Rosetta configuration should be static. Network upgrade shouldn't change its value.
+2. When block_identifier is not specified, the call to /block endpoint should return the tip block.
+3. The online_url and offline_url should be different.
+
+By default, check:spec will verify only Coinbase spec requirements. To verifiy the minimum requirements as well,
+we need to add --all flag, aka rosetta-cli check:spec --all --configuration-file [filepath].`,
```

Comment:
Please see some edits below in the suggestion. I have an additional question below that.

```suggestion
		Long: `Check:spec checks whether a Rosetta implementation satisfies either Coinbase-specific requirements or minimum requirements specified in rosetta-api.org. 
		
By default, check:spec will verify only Coinbase spec requirements. To verify Rosetta's minimum requirements as well, add the --all flag to the check:spec command:
 
rosetta-cli check:spec --all --configuration-file [filepath].
		
The minimum requirements verify whether an API response contains the required fields, and that the fields are correctly formatted with proper values. For example, it would check whether the response of /network/list contains a list of network identifiers.
		
The Coinbase-specific requirements are not documented in rosetta-api.org. However, we highly recommend that your implementation satisfies them. This ensures that, when you want to integrate your asset into the Coinbase platform, you can limit or eliminate implementation issues.
		
Here are a few examples of Coinbase spec requirements:
1. The network_identifier in Rosetta configuration must be static. Network upgrade shouldn't change its value.
2. When block_identifier is not specified, the call to /block endpoint should return the tip block.
3. The online_url and offline_url should be different.

```

Question: Where can we direct users to the rest of the Coinbase spec requirements? Is this something they can find in the rosetta-cli code? Or do we need to direct users to the checklist (which hopefully we can add to Coinbase Cloud soon).

Reply:
@racbc for now the easiest way to check Coinbase spec requirements is to run the check:spec command w/o --all flag. It will print the result of Coinbase spec verification only. They can also find it in the code (check_spec.go). 

@xiaying-peng wrote a internal doc to add all Coinbase spec requirements but it's not visible to external users.

Reply:
Changed long description, thanks!

Reply:
Thanks! I will modify the text then in case anyone wants to look at the code.

Reply:
Thanks!

Reply:
@shiatcb I added additional text to the suggestion above based on our conversation. Please take a look. Thanks!